Thirty/Thirty/Utilities/FirebaseManager.swift

```diff
@@ -333,25 +271,117 @@ class FirebaseManager {
                     UserManager.shared.featuredUsers.append(featuredUser)
                 }
                 UserManager.shared.featuredUsers.sort(by: { $0.username.lowercased() < $1.username.lowercased() })
-                completion(.Success)
+                completion(.success)
             } else {
-                completion(.Failure(THError.unableToGetUsers))
+                completion(.failure(THError.unableToGetUsers))
             }
         }) { (error) in
-            completion(.Failure(error))
+            completion(.failure(error))
         }
     }
+
+    // MARK: Device Token
+
+    func updateDeviceToken() {
+        databaseRef.child("users").child(UserManager.shared.currentUserUsername.lowercased()).updateChildValues(["device-token": TokenUtils.deviceToken])
```

Comment:
We use "device-token" string in multiple places.  We might want to make these childValues constants?

Reply:
yeah there's a lot of low hanging fruit like this in the firebase manager, i think we should deliberate and do all of the models/constants in one sweep. I want to check out leveraging `Codable` for doing a lot of this stuff.